.github/workflows/reusable-acceptance.yml

```diff
@@ -0,0 +1,128 @@
+name: reusable-acceptance
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      additional-flags:
+        required: false
+        type: string
+        default: ""
+      consul-k8s-image:
+        required: false 
+        type: string
+        default: docker.mirror.hashicorp.services/hashicorpdev/consul-k8s-control-plane:latest
+      directory:
+        required: true
+        type: string
+      go-version:
+        required: true
+        type: string
+      gotestsum-version:
+        required: true
+        type: string
+      kind-version:
+        required: false
+        type: string
+        default: "v1.22.4"
+    secrets:
+        CONSUL_ENT_LICENSE:
+          required: true
+
+# Environment variables can only be used at the step level
+env:
+  TEST_RESULTS: /tmp/test-results # path to where test results are saved
+  CONSUL_ENT_LICENSE: ${{ secrets.CONSUL_ENT_LICENSE }} 
+
+jobs:
+  a:
+    runs-on: ubuntu-latest
+    strategy: 
+      matrix:
+        include: # I am really sorry for this but I could not find a way to automatically split our tests into several runners. For now, split manually.
```

Comment:
Github Actions do not have a way to split tests like CircleCI does with `circlecI test split` so I had to split the tests manually so that they would run in parallel. 

This reusable workflow is not called right now but it is in place for the summer when larger VMs are available.